catkit/hardware/iris_ao/iris_ao_controller.py

```diff
@@ -79,10 +88,18 @@ def send_data(self, data):
 
     def _open(self):
         """Open a connection to the IrisAO"""
-        self.instrument = self.instrument_lib.Popen([self.full_path_dm_exe, self.disable_hardware],
+        cmd = [self.full_path_dm_exe, self.disable_hardware]
+        if self.path_to_ini_files:
+            cmd.append(self.path_to_ini_files)
+        if self.filename_ptt_dm:
+            cmd.append(self.filename_ptt_dm)
```

Comment:
This shouldn't mess up jost, however, it also won't work on jost anyhow as the C++ on jost will just look for this file not in ``filename_ptt_dm`` but in the path staticly built into the C++ source.